src/v2/controllers/FormController.ts

```diff
@@ -356,6 +356,7 @@ export default Controller.extend({
   },
 
   transformIdentifier(formName, modelJSON) {
+    const originalIdentifier = modelJSON.identifier;
```

Comment:
Couldn't we do this in the code above instead?

Side note: this function is confusing because it returns an object but it actually modifies the passed object as well. It's worth update it to either return `void` to make it clear that the passed object is being mutated or it should return a single `string` with the value of transformed identifier.